manifests/smee/deployment.yaml

```diff
@@ -0,0 +1,30 @@
+---
+kind: Deployment
+apiVersion: apps/v1
+metadata:
+  name: smee
```

Comment:
Hm, I wonder, why do you need `smee` manifests in here? What is it used for? Testing? Dev OCP deployment?

Reply:
testing -- the public [smee](https://smee.io/) goes down frequently, so I just deployed my own, I'll put in docs after this pr on how to set up a dev environment and leverage these manifests should the need arise

Reply:
Not an issue, just got me confused without context. That's fine. Le'ts keep it here and have some docs around it later on. :slightly_smiling_face: